### ISSUE TITLE
general comments

### ISSUE THREAD

**Issue by dcherian** [e-marshall/itslive#9]
This issue collects some higher-level comments. See https://github.com/e-marshall/mynewbook/pull/10 for examples.

-----

I noticed this pattern repeats (looping over a index using range)
```python
for granule in range(len(input_dict['features'])):
    if input_dict['features'][granule]['properties']['data_epsg'] == epsg_code:
        pass
```

Python for loops are sweet because you can instead directly iterate over the values to write more readable code. So...
```python
for granule in input_dict['features']:
    if granule['properties']['data_epsg'] == epsg_code:
        pass
```

-----

The following pattern
``` python
out = []
for a in b:
    ...
   out.append(...)
```

is usually a good place to use a list comprehension.

**Comment by e-marshall** [comment]
Thanks! This is good to know, I'll make these changes

**Comment by e-marshall** [comment]
closed with 7931936c7b3312879d799c9b08a00aac90424eb8
